keras_nlp/samplers/beam_sampler.py

```diff
@@ -61,11 +61,12 @@ def next(prompt, state, index):
         index=5,
     )
     print(["".join([int_lookup[i] for i in s]) for s in output.numpy()])
-    # >>> "zzzzzaaaaaaa"
+    # >>> ['zzzzzeeeeeee']
     ```
+
     Return all beams and their probabilities.
     ```python
-    # Use a simple alphabet of lowercase characters to [0, 26).
+    # Use a simple alphabet of lowercase characters with ids in range [0, 25].
     int_lookup = {i: chr(i + ord('a')) for i in range(26)}
     char_lookup = {v: k for k, v in int_lookup.items()}
     batch_size, length, vocab_size = 1, 12, len(int_lookup)
```

Comment:
nit: maybe let's update this example to run to a sequence length of 8, we can keep the output on line 90 to a single line